app/services/purl_and_stacks_service.rb

```diff
@@ -43,11 +43,10 @@ def update(cocina_object:, file_uploads:, version:, version_date:, must_version:
   end
 
   # Delete the PURL and Stacks files.
-  # @param version [String] the version number
-  def delete(version:)
+  def delete
     if versioned_files_enabled? && already_versioned_layout?
       begin
-        versioned_files_service.delete(version:)
+        versioned_files_service.delete
```

Comment:
Do we need this to go to the VersionedFilesService?  This is a full object operation now?

Reply:
In my mind, once we complete migrating then the PurlAndStacksService abstraction can be removed. Having delete within VersionedFilesService aligns with that.